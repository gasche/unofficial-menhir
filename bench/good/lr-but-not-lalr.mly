%token A B C D F
%type <unit> s
%start s

%%

s:
  A e C {}
| A f D {}
| B e D {}
| B f C {}

e:
  F {}

f:
  F {}

