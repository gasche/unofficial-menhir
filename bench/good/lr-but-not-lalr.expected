%start s
%token F
%token D
%token C
%token B
%token A

%type <unit> s
%%

s:
| A e C
    {}
| A f D
    {}
| B e D
    {}
| B f C
    {}

e:
| F
    {}

f:
| F
    {}

%%



